app/blueprints/fund/routes.py

```diff
@@ -93,14 +92,11 @@ def create_fund():
             ),
         )
         add_fund(new_fund)
-        flash(f"""
-            <h3 class="govuk-notification-banner__heading">New grant added successfully</h3>
-            <p class="govuk-body">
-                <a class="govuk-notification-banner__link" href="#">
-                View {form.name_en.data}
-                </a>
-            </p>
-        """)
+        flash_message(
+            message="New grant added successfully",
+            href="#",
```

Comment:
Can we update this to the 'view grant' URL now? I know unrelated, but since we're touching the code and that page exists, might as well fix it here 🙏